client/src/routes/routes.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Route, IndexRoute } from 'react-router';
+import App from '../containers/App';
+import Welcome from '../components/Welcome';
+import SubmitCode from '../components/SubmitCode';
+
+const routes = (
```

Comment:
Does this need to be defined in its own file or can we keep everything in index.js? Just curious.